### ISSUE TITLE
Android SDK size

### ISSUE THREAD

**Issue by whitipet** [Netvent/storyly-mobile#36]
The SDK adds 10mb + to the application size. This is due to the Bundled emoji font. The SDK uses [Emoji Bundled fonts configuration](https://developer.android.com/guide/topics/ui/look-and-feel/emoji-compat#bundled-fonts), is there a real need for this? You can use [Emoji Downloadable fonts configuration](https://developer.android.com/guide/topics/ui/look-and-feel/emoji-compat#downloadable-fonts) as described in the [documentation](https://developer.android.com/guide/topics/ui/look-and-feel/emoji-compat), or allow developers to choose whether they need emoji in the application.

**Comment by whitipet** [comment]
The only thing that helped me was adding aaptOptions to build.gradle.
```
buildTypes {
  release {
    aaptOptions {
      ignoreAssetsPattern '!fonts:!*.ttf:'
    }
  }
}
```
This removes all fonts from the assets folder (even the ones you added). Also if you use the "Emoji Reaction" component the app will start crashing (this is not critical for me).

**Comment by JohannaTekinder** [comment]
Hi,
is there a plan to solve this issue?
Removing all fonts isn't a solution for us.

**Comment by levent-oral** [comment]
Hello, 

We're actively working on changing embedded fonts with downloadable ones. It'll be available with upcoming releases

Thanks,

**Comment by levent-oral** [comment]
Hello, 

We've released [Android SDK 1.8.7](https://github.com/Netvent/storyly-mobile/blob/master/android/RELEASENOTES.md#187). Please check this release for emoji font usage change. 

We'll continue to make improvements on reducing effect to apk size. 

Thanks,

**Comment by whitipet** [comment]
I can confirm that the fonts from emoji are gone (the size has decreased significantly). However, there are other fonts (Vollkorn, Bellota, Poppins, CourierPrime, DancingScript) that can also be done using downloadable fonts. Or they can be removed completely.
